src/partner/partner.service.ts

```diff
@@ -23,4 +23,11 @@ export class PartnerService {
   async fetchPartners(): Promise<PartnerEntity[]> {
     return await this.partnerRepository.find();
   }
+
+  async fetchPartner(name: string): Promise<PartnerEntity> {
```

Comment:
if we use `getSession` `getUser` etc everywhere else, maybe should use `getPartners/getPartner` here too